app/javascript/src/QuickStarts/templates/createMappingRule.yaml

```diff
@@ -0,0 +1,126 @@
+  icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
+  description: |-
+    Associate a call to an endpoint with the created methods and metrics.
+  introduction: |-
+    This quick start guides you through how to associate a call to an endpoint with the created methods and metrics.
+    
+    To learn more about mapping rules, see the [3scale documentation](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.10/html-single/admin_portal_guide/index#adding-mapping-rules-to-methods-and-metrics_methods-and-metrics-for-products-and-backends).
+
+    A mapping rule associates a call to an endpoint with designated methods and metrics for tracking and limiting access to your API. Based on the requests to your API, mapping rules define the metrics or designate the methods for which you want to capture API usage.
+
+    Mapping rules work at the API product and API backend levels.  You can define a new mapping rule at any time.  
+    
+    To learn more about mapping rules both at the product and backend levels, see [How APIcast applies mapping rules for capturing usage of 3scale APIs](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.11/html-single/administering_the_api_gateway/index#how-apicast-applies-mapping-rules-for-capturing-usage-of-threescale-apis_api-gateway-apicast).
+  tasks:
+    - title: Define mapping rules at backend level
+      description: |-
+        The advantage of defining mapping rules at the backend level is that you can add a backend to multiple products.
+
+        Mapping rules at the backend level.
+        
+        - When you add mapping rules to a backend, these are added to all the products bundling this backend.
+        - The mapping rule is evaluated after the mapping rules defined at the product level.
+        - The mapping rule is evaluated only if the traffic is redirected to the same backend the mapping rule belongs to.
+        - The path of the backend for a product is automatically prepended to each mapping rule of the backend bundled to said product. 
+
+
+        ### Prerequisites
+
+        - A backend. To create one, see Create a backend.
+        - A method. Mapping rules are required in your previously created methods, however, they are optional for metrics, To create one, see Create a method.
```

Comment:
'Create a method' is a link, which link to "createMethod.yaml'

Reply:
I've created a link for this one, but I cannot preview it. It shows an error for me. Maybe there is another way to test it other than clicking it from the preview to see if it loads the content for Create a method in the createMethod.yaml file.